front/app/modules/commercial/native_surveys/components/NativeSurveyButtonBar/NativeSurveyButtonBar.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
```

Comment:
Suggestion not blocking: Could re-name this to `index.tsx` since there is no other file in the folder and it is the main folder component. We could also call the folder the NativeSurveyFooter. That would allow us add any footer related work in here.